docker-compose.firehose-fetcher.yml

```diff
@@ -0,0 +1,25 @@
+version: "2"
+services:
+  firehose-fetcher:
+    image: registry.devshift.net/bayesian/firehose-fetcher
+    network_mode: bridge
+    container_name: firehose-fetcher
+    restart: always
+    depends_on:
+     - worker-api
+     - worker-ingestion
+    links:
+     - broker
+     - pgbouncer
+     - minio-s3
+    environment:
+      DEPLOYMENT_PREFIX: "${USER}"
+      FIREHOSE_STREAM_URL: "http://firehose.kklibraries.io/events"
```

Comment:
I think this can be safely removed or commented out - https://github.com/fabric8-analytics/fabric8-analytics-firehose-fetcher/pull/1/files#diff-f3a3bc03e40566a9eadf8823e8168f47R5